tests/test_camera.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+from unittest.mock import patch
+
+import cv2
+import numpy as np
+from pathlib import Path
+
+import sense.camera as camera
+
+video_path = str(Path('tests/resources/test_video.mp4').resolve())
```

Comment:
```suggestion
VIDEO_PATH = str(Path('tests/resources/test_video.mp4').resolve())
```
variable name should be all caps when its a global constant